### PR TITLE
tests: add coverage for verifying inclusion against massifs after the…

### PR DESCRIPTION
**Pull request by robinbryce**
tests for inclusion proofs for massifs after the first. Also do the obvious boundary cases.

### CODE REVIEW

File: tests/verifyincluded/verifyevents_azurite_test.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+					"verify-included",
+				})
+				s.NoError(err)
+				s.ReplaceStdIO() // reset stdin for write & close
+			}
+		})
+	}
+}
+
+/*
```
Comment (robinbryce) [inline tests/verifyincluded/verifyevents_azurite_test.go:102]: stale copy/paste will remove

**Review by henry739 — APPROVED** [review]
